### DISCUSSION TITLE
Discovered: SONOFF Zigbee 3.0 USB Dongle Plus V2 - Zigbee Home Automation

### DISCUSSION THREAD

**Discussion by P1X3L8** [ksjh/silabs-firmware-builder#7] [Q&A] [answered]
Not really an issue but a question. Are we supposed to add this when it's discovered? I have both the Multiprotocol and Z2M addons installed and working, but this comes up after rebooting. 

If I click configure it has the error "Failed to probe the usb device".

If I try to install manually I can get as far as needing the device path and port speed.

Thanks
-PiXEL8

**Comment by ksjh** [comment, marked as answer]
As far as I remember, this Zigbee 3.0 USB Dongle Plus V2 message also appeared when I plugged the ZBDongle-E in the first time, regardless which firmware variant was installed. But this integration essentially tries to install the ZHA (Zigbee Home Automation) with direct access to the hardware. When you use the EmberZNet (NCP) firmware version, you can install this integration as an alternative to zigbee2mqtt, but not both. So, either *zha* or *zigbee2mqtt*, you can decide.

The integration also shows up with the RCP firmware variant, but this will not really work. With the RCP firmware variant, you need to install the Silicon Labs Multiprotocol add-on to access the hardware directly. So, this Zigbee 3.0 USB Dongle Plus V2 ZHA integration will fail to install, and this is correct, since the hardware (USB dongle) needs to be accessed exclusively by the Multiprotocol add-on. So, I suggest to simply ``ignore`` this ``Zigbee 3.0 USB Dongle Plus V2 zha`` integration with the respective button provided, so that it will not show up again later. But also in case of using the RCP firmware variant with the Multiprotocol add-on, you can also select either *zigbee2mqtt* or *zha* to control your zigbee devices.

If you want to use *zigbee2mqtt* or *zha* is your choice, independent of the firmware variant, but you need the Multiprotocol add-on when the dongle uses the RCP variant of the firmware.

To summarize: when using the RCP firmware **or** zigbee2mqtt, please ignore the ``Zigbee 3.0 USB Dongle Plus V2 zha`` integration.

**Comment by P1X3L8** [comment]
Thank you ksjh!
